packages/functional-tests/lib/fixtures/standard.ts

```diff
@@ -54,24 +54,30 @@ export const test = base.extend<TestOptions, WorkerOptions>({
     await testAccountTracker.destroyAllAccounts();
   },
 
-  storageState: async ({ target }, use) => {
+  storageState: async ({ target }, use, testInfo) => {
     // This is to store our session without logging in through the ui
+    const localStorageItems = [
+      {
+        name: '__fxa_storage.experiment.generalizedReactApp',
+        value: JSON.stringify({ enrolled: false }),
+      },
+    ];
+
+    // Temporary fix, only set this flag if the test is not a recovery key promo test
+    // Once this is no longer feature flagged, we can update all our test and remove this
+    if (!testInfo.titlePath.includes('recovery key promo')) {
```

Comment:
Not great, but not terrible? We set the dismiss flag for all functional tests except the recovery key promo tests. There would be a lot of test churn and test conditionals otherwise.

Reply:
Heh yeah, I'm OK with this approach, I guess maybe one small tweak could be to put this test name in a shared string so it's clear where we're referencing it. We should file a follow up issue to remove the feature flag and make whatever tweaks we need here.